src/routes/preHandlers/courses.js

```diff
@@ -39,7 +40,7 @@ exports.authorizeCourseEdit = async (req) => {
 
     const courseTrainerId = course.trainer ? course.trainer.toHexString() : null;
     const courseCompanyId = course.company ? course.company.toHexString() : null;
```

Comment:
Ils ont sorti un truc trop cool récemment : https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-

En gros tu fais : course?.trainer.toHexString() et ça te renvois null si jamais course.trainer n'existe pas.

Reply:
a ouai trop cool !  (je met le lien pour la version js : https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Op%C3%A9rateurs/Optional_chaining)

Reply:
en revanche j'arrive pas a l'utiliser, je sais pas si c'est esLint, le validator de vsCode ou un autre truc qui bloque encore

Reply:
je pense que c'est notre version de node qui n'est pas compatible. d'apres ton lien corentin c'est pour la version 14

Reply:
Probablement EsLint, cet outil est assez neuf et il est pas encore trop apprécié. Je vais me renseigner